modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/AccountCreationModalBody.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayForm, {ClayInput, ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayMultiSelect from '@clayui/multi-select';
+import {fetch} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+const ORGANIZATIONS_ROOT_ENDPOINT = '/o/headless-admin-user/v1.0/organizations';
+const ACCOUNTS_ROOT_ENDPOINT = '/o/headless-admin-user/v1.0/accounts';
+
+const orgUrl = new URL(
+	`${themeDisplay.getPathContext()}${ORGANIZATIONS_ROOT_ENDPOINT}`,
+	themeDisplay.getPortalURL()
+);
+
+fetch(orgUrl.toString())
+	.then((response) => response.json())
+	.then((data) => {
+		organizationData = data.items;
+	});
+
+function capitalizeWord(word) {
+	return word[0].toUpperCase() + word.substr(1).toLowerCase();
+}
+
+let organizationData;
+
+const filterOrganizations = (organizations, accountOrganizations) => {
+	if (!accountOrganizations.length) {
+		return organizations;
+	}
+
+	const accountValues = accountOrganizations.map((item) => item.value);
+
+	return organizations.filter((org) => !accountValues.includes(org.value));
+};
+
+export default function AccountCreationModalBody({accountTypes}) {
+	const [organizationQuery, setOrganizationQuery] = useState('');
+	const [accountOrganizations, setAccountOrganizations] = useState([]);
+	const [accountDescription, setAccountDescription] = useState('');
+	const [accountName, setAccountName] = useState('');
+	const [accountType, setAccountType] = useState(accountTypes[0]);
+	const [accountTaxId, setAccountTaxId] = useState('');
+	const [accountERC, setAccountERC] = useState('');
+
+	const organizations = organizationData.map(({id, name}) => {
+		return {label: name, value: id};
+	});
+
+	const createAccount = (event) => {
+		event.preventDefault();
+
+		const organizationIds = accountOrganizations.map(({value}) =>
+			Number.parseInt(value, 10)
```

Comment:
Why do you parse these as numbers, but then immediately stringify it via JSON.stringify?